python/python/bystro/api/cli.py

```diff
@@ -572,6 +626,14 @@ def main():
     )
     jobs_parser.set_defaults(func=get_jobs)
 
+    query_parser = subparsers.add_parser("query", help="Perform a query")
+    query_parser.add_argument("--dir", default=DEFAULT_DIR, help="Where Bystro API login state is saved")
+    query_parser.add_argument("--query", required=True, help="Query properties")
+    query_parser.add_argument("--size", default=10, type=int, help="How many records, max should be the maximum the server supports (default: 10)")
```

Comment:
"max should be the maximum the server supports " - let's remove this, that is not a concern for users, that was mostly a note for us to consider.